streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -128,9 +128,13 @@
     /** <code>client.id</code> */
     public static final String CLIENT_ID_CONFIG = CommonClientConfigs.CLIENT_ID_CONFIG;
 
+    /** <code>rocksdb.config.setter</code> */
     public static final String ROCKSDB_CONFIG_SETTER_CLASS_CONFIG = "rocksdb.config.setter";
     public static final String ROCKSDB_CONFIG_SETTER_CLASS_DOC = "A Rocks DB config setter class that implements the <code>RocksDBConfigSetter</code> interface";
 
+    /** <code>windowstore.changelog.additional.retention.ms</code> */
+    public static final String WINDOW_STORE_CHANGE_LOG_ADDITIONAL_RETENTION_MS_CONFIG = "windowstore.changelog.additional.retention.ms";
```

Comment:
@guozhangwang sounds good. My only question is: what do we do about validating the `retentionConfigs` ? All of that validation is currently done in `core` in `LogConfig` which we can't import and I don't really want to duplicate it. Do we just restrict it to only the set of valid retention properties from `LogConfig` ,i.e, `retention.ms` and `retention.bytes`